caos.ansible_roles/roles/assert_service_status/tasks/assert-service-Darwin.yaml

```diff
@@ -0,0 +1,5 @@
+---
+
+# TODO
```

Comment:
Is this expected?

Reply:
Kind of. This is what we currently have in the agent, and the scope of the task didn't cover "redo what is missing" 😉